modules/experimental-layers/src/mesh-layer/mesh-layer.js

```diff
@@ -123,6 +128,17 @@ export default class MeshLayer extends Layer {
 
   initializeState() {
     const attributeManager = this.getAttributeManager();
+
+    this.state.matrixData = new Float32Array(this.getNumInstances() * 16);
+    this.state.matrixBuffer = new Buffer(this.context.gl, this.state.matrixData.byteLength);
+
+    this.state.buffers = {
```

Comment:
While this works, let's think a bit...

* Today attributes are "owned" by the AttributeManager, also owning them here in the layer is not ideal.
* We should try to improve the attribute manager API to handle this case.
* Would it be possible to allow an accessor update function to return multiple attributes?